src/logic/calculate.test.js

```diff
@@ -0,0 +1,62 @@
+import calculate from './calculate';
+
+test('AC button clears everything', () => {
+  const obj = {
+    total: 10,
+    next: null,
+    operation: '-',
+  };
+  expect(calculate(obj, 'AC')).toEqual({
+    total: 0,
+    next: null,
+    operation: null,
+  });
+});
+test('Plus sign adds up the total', () => {
+  const obj2 = {
+    total: 0,
+    next: 10,
+    operation: '+',
+  };
+  expect(calculate(obj2, '+')).toEqual({
+    total: '10',
+    next: null,
+    operation: '+',
+  });
+});
+test('Minus sign subtructs from the total', () => {
+  const obj = {
+    total: 0,
+    next: 10,
+    operation: '-',
+  };
+  expect(calculate(obj, '-')).toEqual({
+    total: '-10',
+    next: null,
+    operation: '-',
+  });
+});
+test('Multiplication sign multiplies on the total', () => {
+  const obj = {
+    total: 2,
+    next: 10,
+    operation: 'x',
+  };
+  expect(calculate(obj, 'x')).toEqual({
+    total: '20',
+    next: null,
+    operation: 'x',
+  });
+});
+test('Multiplication sign multiplies on the total', () => {
+  const obj = {
+    total: 100,
+    next: 10,
+    operation: '÷',
+  };
+  expect(calculate(obj, '÷')).toEqual({
+    total: '10',
+    next: null,
+    operation: '÷',
+  });
+});
```

Comment:
- [ ] Great job here, all tests are passing successfully, however, let us add few tests to cover all the tests required 

- should throw 0 error if trying to make a zero division
- should throw Zero division error
- should give the modules of the two operand if the buttonName is %
- should display the given value as next (number to be dislayed)

Reply:
Well done guys